fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -172,24 +147,19 @@ public class SessionVariable implements Serializable, Writable {
 
     public static final long DEFAULT_INSERT_VISIBLE_TIMEOUT_MS = 10_000;
 
-    public static final String ENABLE_VECTORIZED_ENGINE = "enable_vectorized_engine";
```

Comment:
Please not clear these session variables so early. Some user is using them, although it is useless. If you remove them, user will receive error.